autogpt_platform/frontend/src/components/monitor/FlowRunInfo.tsx

```diff
@@ -1,79 +1,160 @@
-import React, { useCallback } from "react";
-import AutoGPTServerAPI, { GraphMeta } from "@/lib/autogpt-server-api";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import AutoGPTServerAPI, {
+  BlockIORootSchema,
+  Graph,
+  GraphMeta,
+  NodeExecutionResult,
+} from "@/lib/autogpt-server-api";
 import { FlowRun } from "@/lib/types";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import Link from "next/link";
 import { Button, buttonVariants } from "@/components/ui/button";
 import { IconSquare } from "@/components/ui/icons";
-import { Pencil2Icon } from "@radix-ui/react-icons";
+import { ExitIcon, Pencil2Icon } from "@radix-ui/react-icons";
 import moment from "moment/moment";
 import { FlowRunStatusBadge } from "@/components/monitor/FlowRunStatusBadge";
+import RunnerOutputUI, { BlockOutput } from "../runner-ui/RunnerOutputUI";
 
 export const FlowRunInfo: React.FC<
   React.HTMLAttributes<HTMLDivElement> & {
     flow: GraphMeta;
     flowRun: FlowRun;
   }
 > = ({ flow, flowRun, ...props }) => {
+  const [isOutputOpen, setIsOutputOpen] = useState(false);
+  const [graph, setGraph] = useState<Graph | null>(null);
+  const [executionResults, setExecutionResults] = useState<
+    NodeExecutionResult[] | null
+  >(null);
+  const [blockOutputs, setBlockOutputs] = useState<BlockOutput[]>([]);
+  const api = useMemo(() => new AutoGPTServerAPI(), []);
+
+  // Fetch graph and execution data
+  useEffect(() => {
+    api.getGraph(flow.id, flow.version).then((graph) => {
+      setGraph(graph);
+    });
+    api.getGraphExecutionInfo(flow.id, flowRun.id).then((executions) => {
+      setExecutionResults(executions);
+    });
+  }, [api]);
+
+  useEffect(() => {
+    if (!graph || !executionResults) {
+      return;
+    }
+
+    // Filter nodes to only get output blocks
+    const outputNodes = graph.nodes.filter(
+      (node) => node.block_id === "363ae599-353e-4804-937e-b2ee3cef3da4",
```

Comment:
There uiType == BlockType.OUTPUT

Reply:
Where? I don't see in in ts `Node` and there's no `BlockType`